src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,286 @@
+    canUseWanderers: boolean,
+    canUseRunaways: boolean,
+    cleeshSafe: boolean,
+    questLocation: Location,
+    requiredItem: Item,
+    maximise: string
+  ) {
+    this.name = name;
+    this.reward = reward;
+    this.accept = accept;
+    this.questStateProperty = questStateProperty;
+    this.questProgressLimit = questProgressLimit;
+    this.questProgressProperty = questProgressProperty;
+    this.canUseWanderers = canUseWanderers;
+    this.canUseRunaways = canUseRunaways;
+    this.cleeshsafe = cleeshSafe;
+    this.questLocation = questLocation;
+    this.requiredItem = requiredItem;
+    this.maximise = maximise;
+  }
+
+  currentQuest(): boolean {
+    if (get(this.questStateProperty) !== "unstarted") {
+      return true;
+    } else {
+      return false;
+    }
+  }
+}
+
+const conspiracyQuests = [
+  new conspiracyQuestData(
+    "Serum",
+    30,
+    true,
+    "questESpSerum",
+    5,
+    "",
+    false,
+    false,
+    false,
+    $location`The Mansion of Dr. Weirdeaux`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+  new conspiracyQuestData(
+    "Clipper",
+    20,
+    true,
+    "questESpClipper",
+    23,
+    "fingernailsClipped",
+    false,
+    true,
+    false,
+    $location`The Mansion of Dr. Weirdeaux`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "EVE",
+    30,
+    true,
+    "questESpEVE",
+    0,
+    "",
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "FakeMedium",
+    30,
+    true,
+    "questESpFakeMedium",
+    0,
+    "",
+    true,
+    true,
+    true,
+    $location`The Secret Government Laboratory`,
+    $item`none`,
+    ""
+  ),
+  new conspiracyQuestData(
+    "Gore",
+    20,
+    true,
+    "questESpGore",
+    100,
+    "goreCollected",
+    true,
+    false,
+    false,
+    $location`The Secret Government Laboratory`,
+    $item`gore bucket`,
+    "10 meat"
+  ),
+  new conspiracyQuestData(
+    "JunglePun",
+    20,
+    false,
+    "questESpJunglePun",
+    11,
+    "junglePuns",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`encrypted micro-cassette recorder`,
+    "mys"
+  ),
+  new conspiracyQuestData(
+    "OutOfOrder",
+    30,
+    true,
+    "questESpOutOfOrder",
+    99,
+    "",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`GPS-tracking wristwatch`,
+    "10 Initiative"
+  ),
+  new conspiracyQuestData(
+    "Smokes",
+    30,
+    true,
+    "questESpSmokes",
+    10,
+    "",
+    false,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+] as const;
+
+export function available(): boolean {
+  return have() || get("_spookyAirportToday");
+}
+
+export function completedOneTimeQuests(): boolean {
+  for (const i in conspiracyQuests) {
+    const quest = conspiracyQuests[i];
```

Comment:
```suggestion
  for (const quest of conspiracyQuests) {
```
While `for... in` iterates over the keys of a js Object, `for ... of` iterates over the values of an iterable